From f6f06d89cffa41c7cb49228a1867da328226f36f Mon Sep 17 00:00:00 2001
From: Francois Gaffie <francois.gaffie@intel.com>
Date: Fri, 23 Nov 2012 15:56:28 +0100
Subject: [PATCH 09/21] [PORT FROM MAIN] [AUDIO][RhB] Add Stream Flags key to setParameters

BZ: 70126

Audio Route Manager needs to know the type of stream to take the
appropriate routing decision (eg direct Output, MP3 Offload).
It must be define as key and filled by the policy.

Change-Id: I5d2273971ba8d279f3bbda1db811a39e99d3357f
Origin-Change-Id: Iffbf048a1025f222291ea71115913a3694a3a08a
Signed-off-by: Francois Gaffie <francois.gaffie@intel.com>
Reviewed-on: http://android.intel.com:8080/78397
Reviewed-by: Barthes, FabienX <fabienx.barthes@intel.com>
Tested-by: Barthes, FabienX <fabienx.barthes@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/hardware/audio.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 60e4484..9db16c7 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -107,6 +107,9 @@ __BEGIN_DECLS
 /* Screen state */
 #define AUDIO_PARAMETER_KEY_SCREEN_STATE "screen_state"
 
+/* Stream Flags */
+#define AUDIO_PARAMETER_KEY_STREAM_FLAGS "stream_flags"
+
 /**
  *  audio stream parameters
  */
-- 
1.7.4.1

